modules/runner/src/integrationTest/java/org/apache/ignite/internal/configuration/ItSslConfigurationValidationTest.java

```diff
@@ -34,6 +35,7 @@
  * Integration test for checking SSL configuration validation.
  */
 @ExtendWith(WorkDirectoryExtension.class)
+@Disabled("https://issues.apache.org/jira/browse/IGNITE-19315")
```

Comment:
Too bad, I expected us to preserve this function.
Do we prioritize this new feature as the next one that you'll be working on?

Reply:
Sure, it has "Critical" priority. Will work on it asap.